pkg/crc/constants/constants.go

```diff
@@ -61,24 +61,31 @@ func GetAdminHelperURL() string {
 	return GetAdminHelperURLForOs(runtime.GOOS)
 }
 
-func defaultBundleForOs(bundleVersion string) map[string]string {
+func defaultBundleForOs(bundleVersion, podmanVersion string, preset crcpreset.Preset) map[string]string {
+	if preset == crcpreset.Podman {
+		return map[string]string{
+			"darwin":  fmt.Sprintf("crc_podman_hyperkit_%s.crcbundle", podmanVersion),
+			"linux":   fmt.Sprintf("crc_podman_libvirt_%s.crcbundle", podmanVersion),
+			"windows": fmt.Sprintf("crc_podman_hyperv_%s.crcbundle", podmanVersion),
```

Comment:
Wondering how redundant this is with the table we now have with the bundle URLs?